composer.json

```diff
@@ -49,7 +49,7 @@
         "composer/xdebug-handler": "^1.3.3",
         "infection/abstract-testframework-adapter": "^0.2.3",
         "infection/codeception-adapter": "^0.3.0",
-        "infection/include-interceptor": "^0.2.0",
+        "infection/include-interceptor": "^0.2.1",
```

Comment:
@infection/core since for now we went for the polyrepo approach, how about going all the way and have PRs on them being reviewed as well?

Reply:
It's fine yes, but there was a PR though